src/icons/arrow-dropdown.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><svg height="48" viewBox="0 0 48 48" width="48" xmlns="http://www.w3.org/2000/svg"><path d="M14 20l10 10 10-10z"/><path d="M0 0h48v48h-48z" fill="none"/></svg>
```

Comment:
The rest of our SVG icons are internally sized correctly, but it wasn't working out initially for this one.  I think we can change the `width` and `height` here, and maybe ditch setting the dimensions in CSS.